src/providers/vod/vod-provider.ts

```diff
@@ -167,34 +240,40 @@ export class VodProvider extends Provider {
         },
         {slideCuePoints: [], chapterCuePoints: []} as {slideCuePoints: Array<KalturaThumbCuePoint>; chapterCuePoints: Array<KalturaThumbCuePoint>}
       );
-      if (slideCuePoints.length && slideCuePoints[0]?.assetId) {
+
+      const firstAssetId = thumbCuePoints.find(thumb => thumb.assetId)?.assetId;
+      if (firstAssetId) {
```

Comment:
@semarche-kaltura please add a comment explaining why we fetch only the first asset di